website/docs/execution/joint.md

```diff
@@ -151,7 +151,7 @@ This file should have one line per sample, plus a header (first) line. The first
 [sample ID requirements](/docs/gs/inputs#sampleids). For an example sample set membership file, refer to
 [this one in the GATK-SV GitHub repository](https://github.com/broadinstitute/gatk-sv/blob/main/inputs/templates/terra_workspaces/cohort_mode/sample_set_membership_1kgp.tsv.tmpl).
 
-## Workflow instructions {#instructions}
+## [Workflow instructions](#instructions)
```

Comment:
Curly brackets define the ID of a section, and you reference that ID by `[*](#id)`. So, here the markdown is parsed as referencing a section rather than defining a it.